### PR TITLE
feat(executor): implement holder recovery

### PR DESCRIPTION
**Pull request by 00nktk**

### CODE REVIEW

**Review by tukan — COMMENTED** [review]
Lgtm

File: executor/src/transactions/holder/parse.rs [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+use common::ethnum::U256;
+use reth_primitives::B256;
+
+use common::evm_loader::account::TAG_STATE;
+use common::evm_loader::{self};
+use common::solana_sdk::account_info::AccountInfo;
+use common::solana_sdk::pubkey::Pubkey;
+
+// #[repr(C, packed)]
+// pub struct Header {
```
Comment (tukan) [inline executor/src/transactions/holder/parse.rs:14]: Is this is a comment or a leftover?

**Review by tukan — APPROVED** [review]
LGTM

File: executor/src/transactions/holder/parse.rs (hunk 1 above)
Comment (00nktk) [reply executor/src/transactions/holder/parse.rs:14]: a comment

**Pull request merged**